
public class Assignment {

	public static void main(String[] args)
	{
		int num1,num2;
		try
		{
			num1=0;
			num2=62/num1;
			System.out.println(num2);
			System.out.println("hey i am at the end of try block");
		}
		catch (Arithmetic Exception e)
		{
			System.out.println("you should not divide a number by zero")
		}
		catch (exception e)
		{
			System.out.println("exception occured");
		}
		System.out.println("i am out of try catch block in java");
		}
		

	}


